wgpu-types/src/instance.rs

```diff
@@ -429,6 +437,47 @@ impl NoopBackendOptions {
     }
 }
 
+#[derive(Clone, Debug, Default, Copy, PartialEq, Eq)]
+/// Selects which presentation system to use on DX12.
+pub enum Dx12PresentationSystem {
```

Comment:
```suggestion
/// Selects which kind of swapchain to use on DX12.
pub enum Dx12SwapchainKind {
```

Sorry for another rename, but I think this is more accurate.